llmfoundry/models/layers/blocks.py

```diff
@@ -41,6 +41,18 @@ def __init__(
                 'attn_uses_sequence_id': False,
                 'alibi': False,
                 'alibi_bias_max': 8,
+                'rope': False,
```

Comment:
is there any reason to not import this default attn config from configuration_mpt?

Reply:
Done, thanks for the suggestion!